### DISCUSSION TITLE
Payment Gateway for Middle East - ✅

### DISCUSSION THREAD

**Discussion by devomman** [inspireui/support#5535] [Working on Features]
Please Add this Payment Gateway for Middle East Below Documents is for Flutter  Integration:
https://myfatoorah.readme.io/docs/flutter

[Kuwait](https://register.myfatoorah.com/businesstype/KWT)

[Saudi Arabia](https://register.myfatoorah.com/businesstype/SAU)

[Bahrain](https://register.myfatoorah.com/businesstype/BHR)

[UAE](https://register.myfatoorah.com/businesstype/ARE)

[Qatar](https://register.myfatoorah.com/businesstype/QAT)

[Oman](https://register.myfatoorah.com/businesstype/OMN)

[Jordan](https://register.myfatoorah.com/businesstype/JOR)

[Egypt](https://register.myfatoorah.com/businesstype/EGY)

**Comment by osman-kocak** [comment]
[Tiko ](https://www.tikokart.com/Kurumsal/Index)

**Comment by ameenbadri7** [comment]
This new payment is added to the latest version. nice
